src/components/OverflowMenuButton/OverflowMenuButton.js

```diff
@@ -0,0 +1,235 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import _ from 'lodash';
+import styled, { ThemeProvider } from 'styled-components';
+import InteractiveElement from '../InteractiveElement';
+import { colors, boxShadows, typography } from '../styles';
+import ArrowDropUpIcon from '../icons/ArrowDropUpIcon';
+import ArrowDropDownIcon from '../icons/ArrowDropDownIcon';
+
+const ActionButtonWrapper = styled(InteractiveElement)`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 36px;
+  width: ${props => props.theme.actionButtonWidth ? props.theme.actionButtonWidth : '88px'};
+  border-radius: 2px 0 0 2px;
+  background: ${(props) => {
+    if (props.theme.actionButtonBackgroundColor && !props.disabled) {
```

Comment:
By passing in a default theme this could be reduced to a single ternary expression
```suggestion
   ${(props) => props.disabled ? props.theme.actionButtonDisabledBackgroundColor : this.props.theme. actionButtonDisabledBackgroundColor}
```